docker-compose.yml

```diff
@@ -33,16 +53,46 @@ services:
         awslogs-group: /ec2/lisk-across-relayer
         awslogs-stream: ec2/lisk.across.relayer.docker.log
     environment:
-      - REDIS_URL=redis://redis:6379
+      - REDIS_URL=redis://redis_relayer:6371
     ports:
       - ${RELAYER_1_API_SERVER_HOST:-0.0.0.0}:${RELAYER_1_API_SERVER_PORT:-3000}:${RELAYER_1_API_SERVER_PORT:-3000}
     healthcheck:
       test: curl --fail http://relayer_1:${RELAYER_1_API_SERVER_PORT:-3000}/healthz
     extra_hosts:
       - "host.docker.internal:host-gateway"
 
+  rebalancer:
+    container_name: rebalancer
+    image: ${AWS_ECR_REGISTRY}/${AWS_ECR_REPOSITORY}:${ACROSS_RELAYER_IMAGE_TAG}
+    platform: linux/amd64
+    depends_on:
+      redis_rebalancer:
+        condition: service_healthy
+    volumes:
+      - ./scripts/lisk/docker/${NETWORK:-mainnet}/containerStartBot_rebalancer.sh:/home/lisk/across-rebalancer/scripts/start.sh
+    networks:
+      - across_rebalancer_network
+    restart: always
+    command: ["/bin/sh", "/home/lisk/across-rebalancer/scripts/start.sh"]
+    logging:
+      driver: awslogs
+      options:
+        awslogs-region: eu-west-3
+        awslogs-group: /ec2/lisk-across-relayer
+        awslogs-stream: ec2/lisk.across.rebalancer.docker.log
+    environment:
+      - REDIS_URL=redis://redis_rebalancer:6372
+    ports:
+      - ${REBALANCER_API_SERVER_HOST:-0.0.0.0}:${REBALANCER_API_SERVER_PORT:-3001}:${REBALANCER_API_SERVER_PORT:-3001}
+    healthcheck:
+      test: curl --fail http://rebalancer:${REBALANCER_API_SERVER_PORT:-3001}/healthz
```

Comment:
This might fail. Please consider the changes from https://github.com/LiskHQ/across-relayer/pull/72.